workers/deployment_manager/ServiceAccountGenerator/Program.cs

```diff
@@ -0,0 +1,63 @@
+using Google.Protobuf.Collections;
+using Google.Protobuf.WellKnownTypes;
+using Improbable.SpatialOS.ServiceAccount.V1Alpha1;
+using System;
+using System.IO;
+
+namespace ServiceAccountGenerator
+{
+
+    class Program
+    {
+        static void Main(string[] args)
+        {
+            if (args.Length < 1)
+            {
+                Console.Out.WriteLine("Usage: <service account token path>");
```

Comment:
Is this a relative or absolute path?

Reply:
Also there appears to be more args? 

```
            var path = args[0].Trim('"');
            var projectName = args[1];
```

Would recommend using a command line lib (Mono.Options or CommandLine)